eng/Packages.Data.props

```diff
@@ -98,7 +98,7 @@
     <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="5.4.0" />
 
     <!-- OpenTelemetry dependency approved for Azure.Monitor.OpenTelemetry.Exporter package only -->
-    <PackageReference Update="OpenTelemetry" Version="1.0.1"  Condition="'$(MSBuildProjectName)' == 'Azure.Monitor.OpenTelemetry.Exporter'" />
+    <PackageReference Update="OpenTelemetry" Version="1.1.0-beta4"  Condition="'$(MSBuildProjectName)' == 'Azure.Monitor.OpenTelemetry.Exporter'" />
```

Comment:
@pakrym what's the recommended practice for integration with the upcoming release of OpenTelemetry?
1. Create a separate branch? (any branching naming rule?)
2. Use main branch for anything that is not yet GA'ed?

Reply:
Using the main branch is OK while the exporter is in preview.